### PR TITLE
2796901: Convert tests

### PR DESCRIPTION
**Pull request by marcoscano**

### CODE REVIEW

File: tests/src/Functional/MediaEntityFunctionalTestBase.php [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    'administer content types',
+    'administer node fields',
+    'administer node form display',
+    'bypass node access',
+  ];
+
+  /**
+   * An admin test user account.
+   *
+   * @var \Drupal\Core\Session\AccountInterface;
```
Comment (dawehner) [inline tests/src/Functional/MediaEntityFunctionalTestBase.php:62]: Nitpick: These typehints could be UserInterface, as this is what happens when you create a user.

File: tests/src/FunctionalJavascript/MediaEntityJavascriptTestBase.php [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+   * Waits and asserts that a given element is visible.
+   *
+   * @param string $selector
+   *   The CSS selector.
+   * @param int $timeout
+   *   (Optional) Timeout in milliseconds, defaults to 1000.
+   * @param string $message
+   *   (Optional) Message to pass to assertJsCondition().
+   */
+  protected function waitUntilVisible($selector, $timeout = 1000, $message = '') {
```
Comment (dawehner) [inline tests/src/FunctionalJavascript/MediaEntityJavascriptTestBase.php:106]: That is handy, do you want to suggest this to core maybe?

**Review by dawehner — APPROVED** [review]
I really like this conversion. There is clean readable code in there

**Comment by dawehner** [comment]
Great work!

File: tests/src/FunctionalJavascript/MediaEntityJavascriptTestBase.php (hunk 2 above)
Comment (slashrsm) [reply tests/src/FunctionalJavascript/MediaEntityJavascriptTestBase.php:106]: That would be nice!

**Comment by slashrsm** [comment]
This was committed. See https://drupal.org/node/2796901 for more info. Thank you all!

**Pull request closed without merging**